### ISSUE TITLE
Incompatible with python3

### ISSUE THREAD

**Issue by amoghpj** [Fafa87/cellstar#6]
There are tons of incompatibilities with python3. I am trying to fix them but without a better understanding of the whole project, a dumb translation might not be ideal. Any plans for python3 support?

**Comment by berkyalcinkaya** [comment]
Do you have a python3 compatible version of CellStar @amoghpj? I have run in to the same problem.

**Comment by amoghpj** [comment]
Hi, I do not have a functional version of python3 yet. It looks like the conversion is non-trivial, and needs some in-depth understanding of the code. If the author of the project @Fafa87 thinks this is worthwhile, I would be willing to help with the effort, but in the meantime, I have used it successfully in a python2 virtual environment.

**Comment by lnilya** [comment]
I managed to get it to run in Python 3.7, with quite a bit of modifications that "seem" not to affect the segmentation. I think partially it is also related to versions of libraries having different behaviour. Check the included libraries.txt for my version.
However I found the algorithm oddly overdependent on the type of input images and literally nothing I had produced any usable result. 
I couldn't find any documentation on the 25 or so parameters present and it makes it difficult.

 The CellProfiler plugin doesn't work for me either, but yeah the last proven version I think is cell profiler 2 which doesn't even start. 

[cellstart_python3.zip](https://github.com/Fafa87/cellstar/files/7063055/cellstart_python3.zip)
